docs/DeveloperGuide.md

```diff
@@ -296,33 +296,141 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 *{More to be added}*
 
 ### Use cases
-
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
-
-**Use case: Delete a person**
-
-**MSS**
-
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
-
-    Use case ends.
-
-**Extensions**
-
-* 2a. The list is empty.
-
-  Use case ends.
-
-* 3a. The given index is invalid.
-
-    * 3a1. AddressBook shows an error message.
-
-      Use case resumes at step 2.
-
-*{More to be added}*
+(For all use cases below, the **System** is the `WedLog` and the **Actor** is the `user`, unless specified otherwise)
+
+>Use case: UC1 - Add a guest
```

Comment:
The use case header does not seem to stand out. Perhaps use **bold text** or a heading instead?

<img width="456" alt="image" src="https://github.com/AY2324S1-CS2103T-F11-2/tp/assets/61283948/12f2de10-633e-4468-8b06-f35c1a2ba614">